### ISSUE TITLE
Learning rate not going down

### ISSUE THREAD

**Issue by frauzufall** [juglab/N2V_fiji#1]
The learning rate is tracked in the console by printing the value of the tensor `Adam/lr/read` and I have not seen it change. Need to investigate if that's the right tensor to read out and how / where Keras adjusts the learning rate.
